### DISCUSSION TITLE
[blocks]: Form Wizard

### DISCUSSION THREAD

**Discussion by theutkarshdev** [shadcn-ui/ui#4276] [Blocks Request]
### Description

I would like to request a form wizard block for `shadcn/ui`. A form wizard is a multi-step form that guides users through a process by breaking it down into smaller, more manageable steps. This is particularly useful for complex forms that require a lot of information from the user, such as application forms, surveys, or checkout processes.

### Example

https://tailwindcomponents.com/component/multistep-form-with-tailwindcss-and-alpinejs

**Comment by iutx** [comment]
Are these two components helpful to you?
- [shadcn-stepper](https://shadcn-stepper.vercel.app/)
- [auto-form](https://github.com/vantezzen/auto-form)

**Reply by senpro-ingwersenk** [reply]
Can you link the `shadcn-stepper` source, perhaps? Tried to search it via Google and Github but could not find it. The deployment is seemingly down...

**Reply by iutx** [reply]
The `shadcn-stepper` source from [this issue](https://github.com/shadcn-ui/ui/pull/318).
